week1/HomeworkWeek1.py

```diff
@@ -0,0 +1,77 @@
+# Task 1 - Write a python script to print your name and age
+
+name = input("Enter your name")
+age = int(input("Enter your age"))
+
+print("Your name is : ", name, " Your age is : ", age)
```

Comment:
You remember that in JS you can populate a string with variables using template strings as shown here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

The same you can do with Python, but it is called f-strings:
```
name = "Eric"
age = 74
print(f"Hello, {name}. You are {age}.")
```

Reply:
Ah ok . Thanks. I will do.